core/sds-aspect-model-validator/src/main/java/io/openmanufacturing/sds/aspectmodel/shacl/violation/MinLengthViolation.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2022 Robert Bosch Manufacturing Solutions GmbH
+ *
+ * See the AUTHORS file(s) distributed with this work for additional
+ * information regarding authorship.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+package io.openmanufacturing.sds.aspectmodel.shacl.violation;
+
+public record MinLengthViolation(EvaluationContext context, int min, int actual) implements Violation {
+   public static final String ERROR_CODE = "ERR_MIN_LENGTH";
+
+   @Override
+   public String errorCode() {
+      return ERROR_CODE;
+   }
+
+   @Override
+   public String message() {
+      return String.format( "Property %s on %s has length %s, but its length must be greater than or equal to %s",
```

Comment:
General remark on all error message, not just this one: as they are nicely formulated full sentences, they should end with a dot. All our SHACL messages do. Let's keep the style consistent across all error messages.

Reply:
Done